Packages/Tracking Preview/Locomotion/Runtime/Scripts/Teleportation/Teleport Action/Jump Gem Teleport/JumpGem.cs

```diff
@@ -175,8 +175,12 @@ private void Awake()
             }
 
             _jumpGemTeleport = FindObjectOfType<JumpGemTeleport>(true);
-            _audioSource = GetComponentInChildren<AudioSource>(true);
-            _attachmentHand = GetComponentInParent<AttachmentHand>();
```

Comment:
This line has been removed too, did you mean to do that?

Reply:
i absolutely did not - sorry! reverted that now.